site/dev/deploy.sh

```diff
@@ -18,6 +18,10 @@
 
 set -e
 
+remote_name="${1:-apache}"
+
 ./dev/setup_env.sh
 
-mkdocs gh-deploy --no-history --remote-branch=asf-site
+echo " --> Deploy to asf-site branch of remote repository: ${remote_name}"
+
+mkdocs gh-deploy --no-history --remote-branch=asf-site --remote-name ${remote_name}
```

Comment:
maybe we can take out `--no-history` here. 

its confusing to not have history for the asf-site branch https://github.com/apache/iceberg/tree/asf-site